lib/wallet/wallet.js

```diff
@@ -101,11 +101,6 @@ class Wallet extends EventEmitter {
       consensus.MAX_BLOCK_OPENS * bidsPerAuction * bidsCacheSizeInBlocks;
     this.sendBidResults = new LRU(bidsCapacity);
 
-    // TODO (bennett) - make this configurable from options
-    const opensCacheSizeInBlocks = 21;
-    const opensCapacity = consensus.MAX_BLOCK_OPENS * opensCacheSizeInBlocks;
-    this.sendOpenResults = new LRU(opensCapacity);
-
     // TODO (bennett) - make this configurable from options
     const updatesCacheSizeInBlocks = 20;
```

Comment:
Why is the updates cache staying around?

Reply:
The caching logic is staying the wallet.js file still, i just removed opencache, but rest is still in hsd code. The plugin is sort of providing the http handler methods (within wallet http.js) at the moment and it's not providing a way to extend wallet.js yet.
```

    // TODO (bennett) - make this configurable from options
    const bidsCapacity =
      consensus.MAX_BLOCK_OPENS * bidsPerAuction * bidsCacheSizeInBlocks;
    this.sendBidResults = new LRU(bidsCapacity);

    // TODO (bennett) - make this configurable from options
    const updatesCacheSizeInBlocks = 20;
    const updatesPerAuction = 1; // average in March 2020 is ~3.1
    const updateCapacity =
      consensus.MAX_BLOCK_OPENS * updatesPerAuction * updatesCacheSizeInBlocks;
    this.sendUpdateResults = new LRU(updateCapacity);

    this.hsdTransferResults = new LRU(25000);
    this.sendTransferResults = new LRU(25000);
    this.sendFinalizeResults = new LRU(25000);
    this.sendFinishResults = new LRU(25000);
    this.sendRevealResults = new LRU(10000);
```